content/master/concepts/patch-and-transform.md

```diff
@@ -1546,22 +1546,23 @@ patches:
           type: ...
 ```
 
-String transforms support the following 
+String transforms support the following
 {{<hover label="string" line="7">}}types{{</hover>}}
 
 * [Convert](#string-convert)
 * [Format](#string-format)
 * [Regexp](#regular-expression-type)
 * [TrimPrefix](#trim-prefix)
 * [TrimSuffix](#trim-suffix)
+* [Join](#join)
```

Comment:
Can you move this to be the third item in the list so it's alphabetical?

Reply:
Moved it up.